outputs.tf

```diff
@@ -94,14 +94,9 @@ output "global_replication_group_node_groups" {
 # CloudWatch Log Group
 ################################################################################
 
-output "cloudwatch_log_group_name" {
```

Comment:
we can't remove these, that would be a breaking change. but since we are generating more than one, they don't make sense

please keep both of these but add a note of `# TODO - remove at next breaking change`

Reply:
No, this works great - thank you!